cylc/flow/scripts/lint.py

```diff
@@ -426,8 +426,8 @@ def parse_checks(check_args, ignores=None, max_line_len=None, reference=False):
                 regex = r"^.{" + str(max_line_len) + r"}"
                 if reference:
                     msg = (
-                        'line > {max_line_len} characters. Max line length '
-                        'set in pyproject.toml (default 130)'
+                        'line > ``{max_line_len}`` characters. Max line '
+                        ' length set in pyproject.toml (default 130)'
```

Comment:
When generating the reference we don't know what the `max_line_len` is because it's configurable so I think string formatting this will just pull in the default?

Reply:
> When generating the reference we don't know what the `max_line_len` is because it's configurable so I think string formatting this will just pull in the default which wouldn't make sense for the documentation?

Yes